coffee-tool/src/main/java/hu/icellmobilsoft/coffee/tool/gson/JsonUtil.java

```diff
@@ -53,6 +54,77 @@ public class JsonUtil {
 
     private static final Logger LOGGER = Logger.getLogger(JsonUtil.class);
 
+    /**
+     * Converting JSON string to DTO object without throwing exception
+     *
+     * @param <T>
+     *            type of returned object
+     * @param json
+     *            JSON String
+     * @param typeOfT
+     *            type of returned object
+     * @return object
+     * @throws BaseException
+     *             exception
+     */
+    public static <T> T toObjectEx(String json, Type typeOfT) throws BaseException {
+        try {
+            T dto = toObjectGson(json, typeOfT);
+            LOGGER.debug("Converting to Object successful: [" + dto + "]");
+            return dto;
+        } catch (Exception e) {
+            throw new BaseException("Error in converting json [" + json + "] to [" + typeOfT + "]: " + e.getLocalizedMessage(), e);
+        }
+    }
+
+    /**
+     * Convert JSON String to DTO object without throwing exception
+     *
+     * @param <T>
+     *            type of returned object
+     * @param json
+     *            JSON String
+     * @param typeOfT
+     *            type of returned object
+     * @return DTO
+     */
+    public static <T> T toObjectGson(String json, Type typeOfT) {
```

Comment:
what is diff on:
* `JsonUtil.toObjectGson(json, Type)`
* `JsonUtil.toObject(json, Type)`
?

Reply:
one throws an error, the other does not